tests/trainer/optimization/test_optimizers.py

```diff
@@ -50,38 +50,21 @@ def test_optimizer_with_scheduling(tmpdir):
 def test_multi_optimizer_with_scheduling(tmpdir):
     """ Verify that learning rate scheduling is working """
 
-    hparams = EvalModelTemplate.get_default_hparams()
-    model = EvalModelTemplate(**hparams)
-    model.configure_optimizers = model.configure_optimizers__multiple_schedulers
```

Comment:
is there an issue for converting all of the old test utilities to boring model / boring datamodule equivalents? Maybe something we can guarantee for v1.4 to simplify testing?

Reply:
We don't have an issue for it. Just doing it little by little and making sure all new code uses `BoringModel`

> Maybe something we can guarantee for v1.4 to simplify testing?

The benefit is a considerable speedup, more than simplification